blocks/scroll/__button/scroll__button.css

```diff
@@ -0,0 +1,29 @@
+.scroll__button{
+    width: 45px;
+    height: 45px;
+    border: 1.7px solid #499B99;
+    box-sizing: border-box;
+    border-radius: 50%;
+    background-color: transparent;
+    background-image: url('../../../images/scroll__button_prev.svg');
```

Comment:
It's better to place images into the block or element folder if that images used for styles

Reply:
Thanks I don't think we learned of doing that from our previous projects good to know. I moved it to be inside the block since it's used for both previous and next button elements.